sandbox/modules/identity/src/main/java/org/opensearch/identity/rest/action/permission/put/PutPermissionAction.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.rest.action.permission.put;
+
+import org.opensearch.action.ActionType;
+
+/**
+ * This class defines the PutPermissionAction ActionType which corresponds to an action that grant a subject a permission while using Identity.
+ */
+public class PutPermissionAction extends ActionType<PutPermissionResponse> {
+
+    public static final PutPermissionAction INSTANCE = new PutPermissionAction();
+    public static final String NAME = "cluster:admin/permissions/put";
```

Comment:
Hmm, I had not considered that. Maybe we should have a discussion about this on the permission design issue? It seems like it would be more changes than just this one API so I don't want to unilaterally make that decision. I see your point though.

Reply:
I think either way could work I just think it will still have more design ramifications than we expect regardless. I see the benefit to both though.